src/components/List.spec.js

```diff
@@ -0,0 +1,34 @@
+import { mount, shallowMount } from '@vue/test-utils';
+import List from './List';
+import ListItem from './ListItem';
+
+const mountList = shallowMount(List, {
+    headline: "ToDo-List",
+    ToDoList: [
+      {text: "Clean room"},
+      {text: "Finish task1"},
+      {text: "Add task2"}
+    ],
+    add_item_text: ""
+});
+
+describe("List", () => {
+    test('is a Vue instance', () => {
+        const wrapper = mount(List)
+        expect(wrapper.isVueInstance()).toBeTruthy()
+    });
+    test('contains data', () => {
```

Comment:
```suggestion
    test('renders `ListItem`s for each todo', () => {
```
Is probably more telling